frontend/packages/ux-editor/src/testing/complexLayoutMocks.ts

```diff
@@ -0,0 +1,179 @@
+import { ExternalComponent, ExternalFormLayout } from 'app-shared/types/api';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import { IInternalLayout } from '../types/global';
+import { FormComponent } from '../types/FormComponent';
+import { FormContainer } from '../types/FormContainer';
+import { customDataPropertiesMock, customRootPropertiesMock } from './layoutMock';
+import { BASE_CONTAINER_ID } from 'app-shared/constants';
+
+export const component1Id = 'component1';
+export const component2Id = 'component2';
+export const component3Id = 'component3';
+export const component3_1Id = 'component3_1';
+export const component3_2Id = 'component3_2';
+export const component3_1_1Id = 'component3_1_1';
+export const component3_1_2Id = 'component3_1_2';
+export const component3_1_3Id = 'component3_1_3';
+export const component3_1_4Id = 'component3_1_4';
+
```

Comment:
This is test data. It might change on a regular basis, so adding comments here is dangerous because they might soon become outdated. I'll rather try to think of a better name, maybe "layoutWithMultiPageGroup" or something like that.